src/main/resources/templates/db_scripts/schema.json

```diff
@@ -1111,11 +1111,21 @@
       "snippetPath": "updateCallNumberSource.sql",
       "fromModuleVersion": "26.1.0"
     },
+    {
+      "run": "after",
+      "snippetPath": "addDcbInstance.sql",
+      "fromModuleVersion": "26.1.0"
+    },
     {
       "run": "after",
       "snippetPath": "addReindexJobResourceName.sql",
       "fromModuleVersion": "26.1.0"
     },
+    {
+      "run": "after",
+      "snippetPath": "addDcbHolding.sql",
+      "fromModuleVersion": "26.1.0"
+    },
```

Comment:
`fromModuleVersion` have to be changed to the newest

Reply:
updated to 27.0.0